src/System.Net.WebHeaderCollection/pkg/System.Net.WebHeaderCollection.pkgproj

```diff
@@ -8,9 +8,7 @@
     <ProjectReference Include="..\ref\System.Net.WebHeaderCollection.csproj">
       <SupportedFramework>net46;netcore50;netstandardapp1.5</SupportedFramework>
     </ProjectReference>
-    <ProjectReference Include="..\src\System.Net.WebHeaderCollection.csproj">
-      <OSGroup>Windows_NT</OSGroup>
-    </ProjectReference>
+    <ProjectReference Include="..\src\System.Net.WebHeaderCollection.builds" />
```

Comment:
This looks like it made this package RID'less I assume this is intentional but I thought I would call it out just in case.